### PR TITLE
Help page

### PR DESCRIPTION
**Pull request by ynaffit96**
All accordions closed:
Screenshot (7)
Showing only one opened:
Screenshot (8)
All can be open at the same time:
Screenshot (9)

### CODE REVIEW

File: client/pages/_app.js [hunk 1]
```diff
@@ -8,6 +8,7 @@
 import "../styles/globals.css";
 import "../styles/bulma.css";
 import "tippy.js/dist/tippy.css";
+import "../styles/Accordion.module.css"
```
Comment (just1ngray) [inline client/pages/_app.js]: CSS Modules shouldn't be imported globally (unless they have global marked classes, I think marked with `:`).
You should be able to delete this addition 👍

**Review by just1ngray — CHANGES_REQUESTED** [review]
Two very quick things before we merge:
1. Delete your commit to `_app.js` since we don't need to import a css module like that.
2. Make sure you format with `Prettier` before committing!

You don't need to make a new pull request.
```
git checkout helpPage
git pull
> Make your modifications
git commit (FILE) or commit via VS Code's GitHub controller
git push or push via VS Code's GitHub cloud icon
```
Then boom! This pull request will have changes!

**Comment by just1ngray** [comment]
Whoops, and I just noticed this will merge into `master` and not `development`. Maybe you can change it? Otherwise could you please finish my requested changes and then make a new PR into `development`?

Thank ya!

**Comment by ynaffit96** [comment]
Okay I think it's good to go now!

**Comment by ynaffit96** [comment]
NEVERMIND I forgot to use prettier -_-

**Review by just1ngray — APPROVED** [review]
Sweet! 🥇

**Pull request merged**